packages/hash/api/src/graphql/resolvers/blockprotocol/getBlock.ts

```diff
@@ -0,0 +1,20 @@
+import fetch from "node-fetch";
+import { ResolverFn, BlockProtocolBlock } from "../../apiTypes.gen";
+import { GraphQLContext } from "../../context";
+
+export const getBlockProtocolBlocks: ResolverFn<
+  Promise<BlockProtocolBlock[]>,
+  {},
+  GraphQLContext,
+  {}
+> = async () => {
+  const apiKey = process.env.BLOCK_PROTOCOL_API_KEY as string;
```

Comment:
Rather than casting this I would check if it exists and throw an error if it doesn't